raiden/transfer/views.py

```diff
@@ -526,32 +526,24 @@ def secret_from_transfer_task(
     return transfer_state.transfer_description.secret
 
 
-def get_transfer_role(chain_state: ChainState, secrethash: SecretHash) -> Optional[str]:
+def get_transfer_role(chain_state: ChainState, node_address: AddressHex, secrethash: SecretHash) -> Optional[str]:
     """
     Returns 'initiator', 'mediator' or 'target' to signify the role the node has
     in a transfer. If a transfer task is not found for the secrethash then the
     function returns None
     """
-    task = chain_state.payment_mapping.secrethashes_to_task.get(secrethash)
+    task = chain_state.payment_mapping[node_address].secrethashes_to_task.get(secrethash)
     if not task:
         return None
     return role_from_transfer_task(task)
 
 
-def get_transfer_secret(chain_state: ChainState, secrethash: SecretHash) -> Optional[Secret]:
+def get_transfer_secret(chain_state: ChainState, node_address: AddressHex, secrethash: SecretHash) -> Optional[Secret]:
```

Comment:
why are these `AddressHex` types instead of `Address` types?

Reply:
They are basically the same. This is inherited from the raiden codebase, i found useful to use AddressHex because is a hexadecimal string, so the class name is more representative

Reply:
shouldn't we pick one and use the same one everywhere?